docs(Spec and add_model_instructions.md): Added docs and changed the guide on adding a new model

gaiborjosue opened the pull request:
This will create a new folder named docs, which contains two files: spec.yaml and spec_file.md. These include a template for the spec.yaml file and a guide on adding/modifying its information.

Also, I modified the add_model_instructions.md adding the explanation on how to test the model locally before creating a pull request to the principal repository.

satra commented:
@gaiborjosue - this PR has a bunch of unrelated changes (see pyc files) that should not be added. i just updated the `.gitignore`. so please fetch the current repo so that future commits don't add them.

gaiborjosue commented:
> @gaiborjosue - this PR has a bunch of unrelated changes (see pyc files) that should not be added. i just updated the `.gitignore`. so please fetch the current repo so that future commits don't add them.

Hello, yes my bad. I removed that files and updated the fork right now.

hvgazula reviewed (approved):
Looks good to me.
